ts/receiver/dataMessage.ts

```diff
@@ -240,6 +243,15 @@ export async function handleSwarmDataMessage(
           sentAt: sentAtTimestamp,
         });
 
+  if (expireUpdate && !isEmpty(expireUpdate)) {
+    if (isSyncedMessage) {
+      // TODO handle sync messages expiring separately?
+      window.log.debug('WIP: Sync Message dropping');
+    } else {
+      msgModel = handleExpireUpdate(convoToAddMessageTo, msgModel, expireUpdate);
+    }
+  }
+
```

Comment:
anything should be done for that TODO here ? 
      // TODO handle sync messages expiring separately?

Reply:
If my memory serves correctly this is where we would have used synced expiries before we changed our approach to user config. I think we need to make logic for handling disappearing message updates (using your branch) and then letting regular disappearing messages that are sync messages be processed by the pipeline as normal.

Reply:
ok all good. We will check this once merged.
Can you just remove the uneeded  ` expireUpdate ` check in `  if (expireUpdate && !isEmpty(expireUpdate)) {`

Reply:
Fixed